src/neo-vm/ExecutionEngine.cs

```diff
@@ -8,7 +8,7 @@ namespace Neo.VM
 {
     public class ExecutionEngine : IDisposable
     {
-        private readonly IScriptTable table;
+        protected readonly IScriptTable table;
```

Comment:
So we don't need `protected` any more, right?

Reply:
👍